src/AwsCryptographicMaterialProviders/Keyring.dfy

```diff
@@ -19,19 +19,19 @@ module
     method OnEncrypt(input: Crypto.OnEncryptInput)
       returns (res: Result<Crypto.OnEncryptOutput, Crypto.IAwsCryptographicMaterialProvidersException>)
       ensures res.Success?
-      ==>
-        && Materials.EncryptionMaterialsTransitionIsValid(
-          input.materials,
-          res.value.materials
-        )
+              ==>
+                && Materials.EncryptionMaterialsTransitionIsValid(
+                     input.materials,
+                     res.value.materials
+                   )
```

Comment:
Err... I would understand 2 spaces after `ensures`, but why 8?

Reply:
It's because `==>` aligns with its premise, and the conclusion is relatively indented by two spaces.
To avoid the big indentation, add a newline after "ensures" so that the hypothesis of your implication is on its own line.
Would you like me to apply this change manually?